<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saoma</class>
 <widget class="QMainWindow" name="saoma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>saoma</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="cameraCBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>93</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>631</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="cameraView"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
